core/src/Cabana_DeepCopy.hpp

```diff
@@ -241,7 +241,7 @@ inline void deep_copy(
 /*!
   \brief Fill an AoSoA with a tuple.
 
-  \param slice The AoSoA to fill.
+  \param aosoa The AoSoA to fill.
```

Comment:
Yes I know - unrelated. But I saw it and it needed to be fixed.

Reply:
looks good